index.html

```diff
@@ -10,74 +10,112 @@
 </head>
 
 <body>
-  <header>
-    <h1>Jelena de la Sauce</h1>
-    <nav class="main-nav">
-      <a href="#contact" class="button">Contact</a>
+  <header class="container-head">
+    <h1 class="child-head-name">Jelena de la Sauce</h1>
+    <nav class="child-head-nav main-nav">
+      <a href="#contact"  class="button">Contact</a>
+      <a href="#about" class="button">About</a>
       <a href="#biography" class="button">Biography</a>
     </nav>
   </header>
 
-  <figure>
-    <img src="https://media.istockphoto.com/photos/view-from-above-of-home-office-desk-with-laptop-flowers-and-leaves-on-picture-id1030888824?b=1&k=20&m=1030888824&s=170667a&w=0&h=PdSVMCNoVesDaB53lOtkq8Vkjq7wgrSzRaKY0ElbL3Q=" alt="" width="400" height="250" class="coverimg centerimg"> 
-    <figcaption>I am a well organized and communicative Web Development student, who is graduating the boothcamp at neuefische end of March 2022. I am higly motivated to join a young, creative and inspiring team.</figcaption>
+  <summary class="container-sum">
+  <figure class="child-sumpic">
+    <div>
+    <img src="https://media.istockphoto.com/photos/view-from-above-of-home-office-desk-with-laptop-flowers-and-leaves-on-picture-id1030888824?b=1&k=20&m=1030888824&s=170667a&w=0&h=PdSVMCNoVesDaB53lOtkq8Vkjq7wgrSzRaKY0ElbL3Q=" alt="" width="auto" height="auto" id="coverimg"> 
+      <figcaption>This is not the desk of Jelena. :-)</figcaption>
+    </div>
   </figure>
 
   <hr />
```

Comment:
Nice! Had to google it, but sounds like an elegant & easy thing!

Reply:
unfortunately the picture is not really flexible / wrapping - still need to work on this one.
Might also be the picture itself, i am honestly not quite sure and confident about what is working and what is not.
Thank you Maggie! :-)